src/createSession.js

```diff
@@ -36,12 +68,18 @@ export default function createSession({
     }
   };
 
+  let onSessionConnected = (event) => {
+    logAnalyticsEvent(apiKey, sessionId, 'react_connected', event.target.connection.connectionId);
+  };
+
   let eventHandlers = {
     streamCreated: onStreamCreated,
     streamDestroyed: onStreamDestroyed,
+    sessionConnected: onSessionConnected,
   };
 
   let session = OT.initSession(apiKey, sessionId);
+  logAnalyticsEvent(apiKey, sessionId, 'react_initialize');
```

Comment:
It makes sense to have the list of actions as constants. Maybe, it has not impact now,  but if we have more in the future, it could be useful to maintain the code.

Reply:
At this point, there are only two, but I can improve it in a future PR when we have more along with variation. Since the JS SDK is actually logging underneath this library, I didn't want to add too many more logs.